Multisig/UI/Settings/Passcode/PasscodeSettingsViewController.swift

```diff
@@ -119,9 +115,24 @@ class PasscodeSettingsViewController: UITableViewController {
                 (section: .single, rows: [.usePasscode])
             ]
         }
+        updateLockValues()
         tableView.reloadData()
     }
 
+    func updateLockValues() {
+        if AppSettings.passcodeOptions.contains(.useBiometry) {
```

Comment:
Why not .passcodeAndUserPresence? This func could use some explanatory comments.

Reply:
Because it is a temporary change right now. This screen still works with the old authentication mechanism

Reply:
will add the comments when I'll rewrite parts of this class to use Security Center